File: app/src/main/java/com/lcl6/cn/basedialog/mvp/model/DaggerScopModel.java
package com.lcl6.cn.basedialog.mvp.model;

import java.util.List;

/**
 * Created by liancl on 2017/8/31.
 */

public interface DaggerScopModel {
    void getData(LoadCompleteListener listener);
    interface LoadCompleteListener{
        void onComplete(List<String> list);
    }
}

File: app/src/main/java/com/lcl6/cn/basedialog/mvp/model/DaggerScopModel.kt
package com.lcl6.cn.basedialog.mvp.model

/**
 * Created by liancl on 2017/8/31.
 */

interface DaggerScopModel {
    fun getData(listener: LoadCompleteListener)
    interface LoadCompleteListener {
        fun onComplete(list: List<String>)
    }
}
